src/test/kotlin/se/uulm/snowballr/backend/validation/ProjectValidatorTest.kt

```diff
@@ -327,4 +327,45 @@ class ProjectValidatorTest {
             assertInvalidResult<BlankField>(result)
         }
     }
+
+    inner class InformationGetRequest {
+        private val validId = UUID.randomUUID().toString()
+        private fun getRequest(projectId: String, paths: List<String>?) = Project.Information.Get
+            .newBuilder()
+            .setProjectId(projectId.toString())
```

Comment:
`projectId` is already a string.
```suggestion
            .setProjectId(projectId)
```